app/api/helpers/order.py

```diff
@@ -134,7 +135,16 @@ def create_onsite_attendees_for_order(data):
     del data['on_site_tickets']
 
 
-def calculate_order_amount(tickets, discount_code):
+def calculate_order_amount(data):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [calculate_order_amount is too complex (18)](https://app.codacy.com/app/fossasia/open-event-orga-server/pullRequest?prid=4592373)